Documentation/ApiOverview/Environment/Index.rst

```diff
@@ -46,10 +45,12 @@ For projects without Composer setup, this is equal to :ref:`Environment-public-p
 getPublicPath()
 ---------------
 
-The environment provides the path to the public web folder with
-:file:`index.php` for the TYPO3 frontend. This was previously :php:`PATH_site`.
-For projects without Composer setup, this is equal to :ref:`Environment-project-path`.
+The method provides the path to the :ref:`public web folder <directory-public>`
+with :file:`index.php` for the TYPO3 frontend.
```

Comment:
What is a TYPO3 frontend? Yes I know, it was already there and can be part of another PR.

Reply:
I think we are mentioning Frontend and Backend all over TYPO3 explained...